extensions/amp-ad-exit/0.1/amp-ad-exit.js

```diff
@@ -100,9 +100,31 @@ export class AmpAdExit extends AMP.BaseElement {
     if (target.vars) {
       for (const customVar in target.vars) {
         if (customVar[0] == '_') {
-          vars[customVar] = () =>
-              args[customVar] || target.vars[customVar].defaultValue;
-          whitelist[customVar] = true;
+          const vals =
```

Comment:
Can we add some comments here?  Its rather difficult to read.  Indicating the expected url replacement format would be great.